README.md

```diff
@@ -122,7 +122,7 @@ RUN set -x \
 	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini.asc" -o /usr/local/bin/tini.asc \
 	&& export GNUPGHOME="$(mktemp -d)" \
```

Comment:
Yeah, that's a good point -- I was going to update that separately, but might as well include it here too.